integration-cli/docker_cli_images_test.go

```diff
@@ -18,6 +18,40 @@ func (s *DockerSuite) TestImagesEnsureImageIsListed(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestImagesEnsureImageWithTagIsListed(c *check.C) {
+	_, err := buildImage("imagewithtag:v1",
+		`FROM scratch
+		MAINTAINER dockerio1`, true)
+	c.Assert(err, check.IsNil)
+
+	time.Sleep(time.Second)
```

Comment:
I did the same as the other tests (like ``TestImagesOrderedByCreationDate`` for example) that were doing that. I'm not quit sure about the reason.. It seems it was already there when these tests were in ``integration``. 
@cpuguy83 I could test without it if it works well and then *maybe* remove it for the other tests (in another PR then) if we think it's not needed at all :sweat_smile:.